core/signer/signer.go

```diff
@@ -145,7 +157,7 @@ func (s *Signer) applyMutations(mutations []*tpb.SignedKV, leaves []*trillian.Ma
 }
 
 // CreateEpoch signs the current map head.
-func (s *Signer) CreateEpoch(ctx context.Context) error {
+func (s *Signer) CreateEpoch(ctx context.Context, alwaysCreateNewEpoch bool) error {
```

Comment:
Done (in https://github.com/google/keytransparency/pull/593/commits/5987035a462e7f080285f51e879b117ae5e2b2b1)